parastell/parastell.py

```diff
@@ -291,6 +292,112 @@ def export_source_mesh(self, filename='source_mesh', export_dir=''):
             export_dir=export_dir
         )
 
+    def construct_nwl_build(
```

Comment:
This seems identical to `construct_invessel_build` but with tweaked/customized parameters

Reply:
It's similar, but there's some extra work to change `radial_build` such that it contains only the `nwl_geom` component.

Reply:
Maybe I misunderstood our slack exchange, but I thought we were going to still try to streamline this to just use `construct_invessel_build` instead of a separate method?

Reply:
Maybe there was some miscommunication. Inside the `parastell` function, I can basically copy/paste what's in this method (and same for the export method) under the NWL conditional, or I can implement what I discussed in [this comment](https://github.com/svalinn/parastell/pull/100/files#r1586396305).